### PR TITLE
188 second view

### LINKED ISSUE #188: Ability to look at two fully processed images
It would be very beneficial to open two different (.btrgb) files in order to compare them. 

Question:
Is this just the calibration report page, or the report and the image viewer pages?

AC:
- Should create a button somewhere to allow user to open another .btrgb file.
- Containing the UI of the two reports in one window may not be sufficient, in which case open the report in a new window.
- New window should have same functionality as the (reports/image viewer) page(s?)

### PR DESCRIPTION
**Pull request by rfl8484**
Resolves #188 
Added functionality for user to create a second window for the application. Secondary window has full functionality as the original. Both frontend operate independently but share a backend api request handler. 

Home Page, added "Create Another Window" button
image

Calibration Report Page, added "View Another Report" button in header
image

Implementation decision reasoning:
I found that creating a second window with full functionality was easier than creating a secondary view that *only* contained the "Calibration Reports" page. Tried multiple approaches and either the styling would get messed up, the child window wouldn't be able to talk to the backend, which is needed for viewing the reports, or the child window would have an additional menu bar. Also opted to create a new window, as opposed to tabs that the user clicks back and forth between.

### CODE REVIEW

**Review by greco380 — APPROVED** [review]
looks great!

File: frontend/src/renderer/Svelte.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 });
 
+
+// const app = new App({
+// 	target: document.body,
+// 	// props: {
+// 	// 	name: 'world'
+// 	// }
+// });
+
```
Comment (jga9325) [inline frontend/src/renderer/Svelte.ts:14]: Can this commented out code be deleted if it is not being used anymore?

File: frontend/src/renderer/pages/Home.svelte [hunk 2]
```diff
@@ -8,14 +8,26 @@
...
+  
+  
+	
+  
+  const openNewWindow = () => {
+    window.electron.openNewWindow();
+  }
+
+ 
+
```
Comment (jga9325) [inline frontend/src/renderer/pages/Home.svelte:30]: Any reason for all of this extra space?

File: frontend/src/renderer/Svelte.ts (hunk 1 above)
Comment (rfl8484) [reply frontend/src/renderer/Svelte.ts]: fixed

File: frontend/src/renderer/pages/Home.svelte (hunk 2 above)
Comment (rfl8484) [reply frontend/src/renderer/pages/Home.svelte]: nope, it is removed

**Pull request merged**